src/styles.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=Gochi+Hand&family=Roboto+Condensed:wght@300&display=swap');
+
+body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: 'Roboto Condensed', sans-serif;
+}
+
+h1 {
+    font-size: 50px;
+    font-family: 'Gochi Hand', cursive;
+    text-align: center;
+}
+
+.world-map {
+    width: 1000px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.flag {
+    width: 40px;
+    margin: 0 20px 0 0;
+    display: inline-block;
+}
+
+#country-list {
+    list-style-type: none;
+    font-weight: bold;
+    column-count: 3;
+    max-width: 1000px;
+    border-radius: 5px;
+    border: 1px solid lightgrey;
+    padding: 40px;
+}
+
+.population {
+    margin: 0 0 12px 0;
+    font-size: 12px;
+}
+
+.region.blue {
+    color: blue;
+}
+
+.region.green {
+    color: green;
+}
+
+.region.red {
+    color: red;
+}
+
+.region.yellow {
+    color: yellow;
+}
+
+.region.purple {
+    color: purple;
+}
+
+.region.default {
+    color: black;
+}
```

Comment:
het ziet er fantastisch uit Habibe! Echt prachtig hoe je de lijst eronder in Css hebt opgelost. Ik was zelf aan het worstelen en ben jaloers hoe mooi het eruit ziet. Verder niets toe te voegen. Bagus sekali!!!!

Reply:
Heel erg bedankt! Terima kasih :)